lib/MusicBrainz/Server/Controller/WS/js.pm

```diff
@@ -626,6 +626,22 @@ sub entities : Chained('root') PathPart('entities') Args(2)
         $c->model('ArtistCredit')->load(values %{$results});
     }
 
+    if ($ENTITIES{$type_name}{type}) {
+        $c->model(type_to_model($type_name) . 'Type')->load(
+            values %{$results},
```

Comment:
Can we store `my @entities = values %{$results};` above the artist_credits stuff above now that it's repeated 5 times?